src/nomos/agent_tests/Unit/test_nomos_gap.c

```diff
@@ -23,67 +27,102 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #include "nomos_gap.h"
 
-void test_collapseInvisible() {
+/**
+ * \brief Test for collapseInvisible()
+ * \test
+ * -# Create a string with INVISIBLE characters
+ * -# Call collapseInvisible()
+ * -# Check if the string is collapsed with getPairPosOff()
+ */
+void test_collapseInvisible()
+{
   char* buf, *fer;
   GArray* po;
-  buf=g_strdup_printf("\377abc\377\377de\377\377fg\377hi");
-  fer=g_strdup(buf);
+  buf = g_strdup_printf("\377abc\377\377de\377\377fg\377hi");
```

Comment:
Unnecessary reformatting (Formatting should only be done when the line is actually modified).